build.gradle.kts

```diff
@@ -31,6 +32,7 @@ version = "2.0.2"
 description = "A web application to generate Vert.x projects"
 
 val vertxVersion = "3.6.3"
+val junitJupiterVersion = "5.4.1"
 
 dependencies {
   implementation("io.vertx:vertx-web:${vertxVersion}")
```

Comment:
If you set `vertxVersion` in the vertx plugin extension configuration, then you can skip the version in dependencies from the Vert.x stack. It uses the Nebula plugin under the hood.